src/Dfc.CourseDirectory.Functions/local.settings.json

```diff
@@ -3,5 +3,8 @@
   "Values": {
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
     "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+  },
+  "Host": {
```

Comment:
I've added this so that manual trigger functions can be executed via the admin HTTP endpoint (`POST http://localhost:7071/admin/functions/{FunctionName}`) without having to reconfigure it every time.